### ISSUE TITLE
URL hard coded is not reachable 

### ISSUE THREAD

**Issue by jakejasey** [zackelia/ghidra-dark#10]
The URL https://bintray.com/jformdesigner/flatlaf/download_file?file_path=com/formdev/flatlaf/0.4 is returning a forbidden. 

Cannot install the theme while this is the issue.

**Comment by jakejasey** [comment]
Sorry, 

This url https://dl.bintray.com/jformdesigner/flatlaf/com/formdev/flatlaf/0.43/flatlaf-0.43.jar

**Comment by lyellread** [comment]
I have this issue as well. 

```python
python3 install.py --path /opt/ghidra/
Using Ghidra installation path: /opt/ghidra/
Detected Ghidra v9.2.2
Downloading FlatLaf
Traceback (most recent call last):
  File "install.py", line 61, in <module>
    response = urllib.request.urlopen(flatlaf_url)
  File "/usr/lib/python3.8/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.8/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.8/urllib/request.py", line 640, in http_response
    response = self.parent.error(
  File "/usr/lib/python3.8/urllib/request.py", line 563, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.8/urllib/request.py", line 502, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.8/urllib/request.py", line 755, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.8/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.8/urllib/request.py", line 640, in http_response
    response = self.parent.error(
  File "/usr/lib/python3.8/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.8/urllib/request.py", line 502, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.8/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden

```

**Comment by lyellread** [comment]
Appears to be a similar site that hosts this. I cannot attest to it's legitimacy or safety. https://repo1.maven.org/maven2/com/formdev/flatlaf/0.43/flatlaf-0.43.jar

**Comment by zackelia** [comment]
Fixed in #11
